ios/Makefile

```diff
@@ -1,9 +1,18 @@
-archive_path = PolyPod.xcarchive
+archive_path = ./build/PolyPod.xcarchive
+ipa_path = ./build/PolyPod.ipa
 project_path = PolyPodApp/PolyPod.xcodeproj
 scheme_name = PolyPod
 export_options_plist = exportOptions.plist
 destination = 14.4
 
+# Github secrets
```

Comment:
See my comment above, this destionation should fail with:

```
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ platform:iOS Simulator, OS:14.4, name:iPhone 11 }

	The requested device could not be found because no available devices matched the request.

	Available destinations for the "PolyPod" scheme:
		{ platform:iOS Simulator, id:6404EB78-56BC-49FC-A43F-85C221366C1A, OS:15.2, name:iPad (7th generation) }
		{ platform:iOS Simulator, id:7A3911A5-9DCA-4D03-A6FC-D65A970C9343, OS:15.2, name:iPad (8th generation) }
		{ platform:iOS Simulator, id:8E4659AD-CDD1-4921-B0D8-82D489A9A6AF, OS:15.2, name:iPad (9th generation) }
		{ platform:iOS Simulator, id:9F2E049A-6CF7-4160-8C97-FAC7D9C424BC, OS:15.2, name:iPad Air (3rd generation) }
		{ platform:iOS Simulator, id:F12CCA98-98DC-495D-BFBA-EE4C8BCDB7BA, OS:15.2, name:iPad Air (4th generation) }
		{ platform:iOS Simulator, id:8701D7FB-AE39-4715-A239-1A86EF99AB2D, OS:15.2, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:28D88473-A175-43A5-A72D-E6F3662500CC, OS:15.2, name:iPad Pro (11-inch) (2nd generation) }
		{ platform:iOS Simulator, id:FAF81860-D900-4749-BE7B-0940248029EE, OS:15.2, name:iPad Pro (11-inch) (3rd generation) }
		{ platform:iOS Simulator, id:C262C7C7-D2AF-46CB-AA05-1E1B405FD56F, OS:15.2, name:iPad Pro (12.9-inch) (4th generation) }
		{ platform:iOS Simulator, id:476014E5-601D-4B71-9B36-6D73262FF618, OS:15.2, name:iPad Pro (12.9-inch) (5th generation) }
		{ platform:iOS Simulator, id:98E3E6D4-E90E-479A-9E3C-4B69A0649BC2, OS:15.2, name:iPad mini (6th generation) }
		{ platform:iOS Simulator, id:B2216284-A5C2-4D1D-A627-E03F0AB50D71, OS:15.2, name:iPhone 8 }
		{ platform:iOS Simulator, id:10921B42-C38C-45C6-8686-EF47AA4F7630, OS:15.2, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:55F3F80C-142B-45F2-A979-FF4BD3504487, OS:15.2, name:iPhone 11 }
		{ platform:iOS Simulator, id:95DFAB6F-4679-4169-A364-A1B70522FC81, OS:15.2, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:C4E97DBA-4BE3-40AD-8772-C8E49B36BCDD, OS:15.2, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:CA7669C6-9105-49DF-98F6-B2FCA30A6C12, OS:15.2, name:iPhone 12 }
		{ platform:iOS Simulator, id:4A3E628A-693B-44C2-9130-50E83E2EAD12, OS:15.2, name:iPhone 12 Pro }
		{ platform:iOS Simulator, id:C3DE85CF-1C0E-490A-A9CB-CB475D7D4279, OS:15.2, name:iPhone 12 Pro Max }
		{ platform:iOS Simulator, id:41A4D308-3B02-4459-9C39-0ADEC1CC8329, OS:15.2, name:iPhone 12 mini }
		{ platform:iOS Simulator, id:BBCF8E92-4628-40EC-A3C5-BC254D70490F, OS:15.2, name:iPhone 13 }
		{ platform:iOS Simulator, id:9118D43C-E8F3-4A99-97CA-0B62639C6255, OS:15.2, name:iPhone 13 Pro }
		{ platform:iOS Simulator, id:2EBC4D33-0756-4F7C-A995-1AF2387DC76B, OS:15.2, name:iPhone 13 Pro Max }
		{ platform:iOS Simulator, id:58017DD0-E536-477F-9709-514F03CF88C0, OS:15.2, name:iPhone 13 mini }
		{ platform:iOS Simulator, id:4817872C-E211-42A1-8377-0C047EAEC1C8, OS:15.2, name:iPhone SE (2nd generation) }
		{ platform:iOS Simulator, id:C4ECB1FF-D89A-49F1-9B3B-4408E7BAD0E0, OS:15.2, name:iPod touch (7th generation) }
```

Reply:
Essentially, we would need to upgrade destination to 15.2. Can try and do it in #662 if you want.

Reply:
Testing it there anyway...